static/index.html

```diff
@@ -38,8 +38,8 @@
                             <th width="150">Assignee</th>
                             <th width="150">Milestone</th>
                             <th width="100">State</th>
+                            <th width="100">Age (in hours)</th>
                             <th>Repo</th>
-                            <th>HTML</th>
```

Comment:
It's a hidden column and no data is rendered. For now, let's keep the column as it is until we find a better solution.